presto-cache/src/test/java/com/facebook/presto/cache/alluxio/TestAlluxioCachingFileSystem.java

```diff
@@ -182,6 +186,55 @@ public void testStress()
         });
     }
 
+    @Test(timeOut = 30_000)
+    public void testCreationFailure()
+            throws Exception
+    {
+        File cacheDir = new File(cacheDirectory.getPath());
+        cacheDir.setWritable(false);
+        CacheConfig cacheConfig = new CacheConfig()
+                .setCacheType(ALLUXIO)
+                .setCachingEnabled(true)
+                .setBaseDirectory(cacheDirectory);
+        AlluxioCacheConfig alluxioCacheConfig = new AlluxioCacheConfig()
+                .setMaxCacheSize(new DataSize(10, KILOBYTE));
+        Configuration configuration = getHdfsConfiguration(cacheConfig, alluxioCacheConfig);
+        try {
+            cachingFileSystem(configuration);
+        }
+        catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Can we check for AccessDeniedException rather printing whole stack trace?

Reply:
updated to expect AccessDeniedException